stroom-query-api/src/main/java/stroom/query/api/v2/ListBuilder.java

```diff
@@ -0,0 +1,37 @@
+package stroom.query.api.v2;
+
+import stroom.util.shared.OwnedBuilder;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * A general purpose list builder, tied to our OwnedBuilder hierarchy allowing us to embed it in the build chain
+ * @param <OwningBuilder> The class of the owning builder
+ * @param <ListedPojo> The classes being put into the list under construction
+ */
+public class ListBuilder<OwningBuilder extends OwnedBuilder, ListedPojo>
+        extends OwnedBuilder<FlatResult.Builder<OwningBuilder>, List<ListedPojo>, ListBuilder<OwningBuilder, ListedPojo>> {
+    private final List<ListedPojo> childValues = new ArrayList<>();
+
+    /**
+     * Add values to our list
+     * @param values The values to add to our list
+     * @return This builder, allowing method chaining
+     */
+    public ListBuilder<OwningBuilder, ListedPojo> value(final ListedPojo...values) {
```

Comment:
shouldn't this be called addValues, 'value' implies setting a value rather than adding one(or many)